### DISCUSSION TITLE
部署到cloudflare，显示MIME 类型 text/html 问题。

### DISCUSSION THREAD

**Discussion by llds66** [un-pany/v3-admin-vite#228] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/225b3e47-fdb6-484c-a97f-5b5787e35cc7)

我需要在项目中设置什么吗？还是在cloudflare设置？

**Comment by pany-ang** [comment, marked as answer]
1. 检查 build 时对应环境的 VITE_PUBLIC_PATH 变量
2. 检查服务器对静态资源的处理配置

**Reply by llds66** [reply]
谢谢，已解决。
